nilrt_snac/_Configs/_NiauthConfig.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+import subprocess
+
+from nilrt_snac._Configs._BaseConfig import _BaseConfig
+from nilrt_snac._Configs._ConfigFile import _ConfigFile
+
+from nilrt_snac import logger, SNAC_DATA_DIR
+from nilrt_snac.OpkgHelper import opkg_helper
+
+
+class _NiauthConfig(_BaseConfig):
+    def __init__(self):
+        self._config_file = _ConfigFile("/etc/ntp.conf")
```

Comment:
Why is this touched in the NIAuthConfig?

Reply:
Removed